### ISSUE TITLE
missing UnimplementedXxxServer from upstream

### ISSUE THREAD

**Issue by carnott-snap** [gogo/protobuf#628]
golang/protobuf#875 recently added support for an `UnimplementedXxxServer` struct, to offer forward compatibility when implementing the generated `XxxServer` interface[0]. This was released as part of their [`v1.3.2`](https://github.com/golang/protobuf/releases/tag/v1.3.2) release.

I see that gogo/protobuf#191 tracks upstream merges, per the `README.md`, but there has been no discussion since December 2019, so I thought I would try and track this separately.  

0] See grpc ticket: grpc/grpc-go#2318

**Comment by jmarais** [comment]
Hi, Are you referring to this commit:
https://github.com/golang/protobuf/commit/b85cd75de734650db18a99a943fe351d41387800
?

**Comment by carnott-snap** [comment]
Yes, that should be the change. I think this has been merged in now, but I would need to double check.
